charts/hedera-mirror-rest-java/templates/ingress.yaml

```
@@ -19,17 +19,29 @@ metadata:
 spec:
   rules:
   {{- range .Values.ingress.hosts }}
-    - host: {{ $hostname | default .host | quote }}
-      http:
-        paths:
-        {{- range .paths }}
-          - path: {{ .path }}
-            pathType: {{ .type | default "ImplementationSpecific" }}
-            backend:
-              service:
-                name: {{ $fullName }}
-                port:
-                  number: {{ $servicePort }}
+  - host: {{ $hostname | default .host | quote }}
+    http:
+      paths:
+      {{- range .paths }}
+        {{- $pathEnabled := false -}}
+          {{- if eq .path "/api/v1/accounts/(\\d+\\.){0,2}(\\d+|(0x)?[A-Fa-f0-9]{40}|(?:[A-Z2-7]{8})*(?:[A-Z2-7]{2}|[A-Z2-7]{4,5}|[A-Z2-7]{7,8}))/allowances/nfts" }}
+            {{- $pathEnabled = $.Values.gateway.endpoints.accounts.allowances.nfts.enabled -}}
+            {{- else if eq .path "/api/v1/accounts/(\\d+\\.){0,2}(\\d+|(0x)?[A-Fa-f0-9]{40}|(?:[A-Z2-7]{8})*(?:[A-Z2-7]{2}|[A-Z2-7]{4,5}|[A-Z2-7]{7,8}))/airdrops/outstanding" }}
+            {{- $pathEnabled = $.Values.gateway.endpoints.accounts.airdrops.outstanding.enabled -}}
+            {{- else if eq .path "/api/v1/accounts/(\\d+\\.){0,2}(\\d+|(0x)?[A-Fa-f0-9]{40}|(?:[A-Z2-7]{8})*(?:[A-Z2-7]{2}|[A-Z2-7]{4,5}|[A-Z2-7]{7,8}))/airdrops/pending" }}
+            {{- $pathEnabled = $.Values.gateway.endpoints.accounts.airdrops.pending.enabled -}}
+            {{- else if eq .path "/api/v1/topics/(\\d+\\.){0,2}\\d+$" }}
+            {{- $pathEnabled = $.Values.gateway.endpoints.topics.enabled -}}
+          {{- end }}
+          {{- if $pathEnabled }}
```

We should omit existing paths and not repeat regexes. We can add a `condition` to each path. It should default to true if omitted.

```suggestion
  {{- if or (empty .condition) (eq (tpl .condition $) "true") }}
```

Then in values.yaml
```yaml
        - path: '/api/v1/network/stake'
          condition: "{{ .Values.routes.networkStake }}"
```

---

Addressed.